### DISCUSSION TITLE
assignee_email

### DISCUSSION THREAD

**Discussion by huer12** [mozts2005/ZendeskApi_v2#585] [Q&A] [answered]
According to the Zendesk Documentation https://github.com/Speedygeek/ZendeskApi_v2/ the ticket can has the field assignee_email. The Ticket class in this project is not providing this property. 
Is there a particular reason for it or would it be ok to add that in pull request?

**Comment by mozts2005** [comment, marked as answer]
Yes, I would be happy to review a PR adding support.

**Comment by huer12** [comment]
Then here it is:
https://github.com/Speedygeek/ZendeskApi_v2/pull/586
